lang-python/pycares/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=pycares
+PKGSEC=python
+PKGDEP="c-ares cffi idna"
+BUILDDEP="python-build python-installer wheel"
+PKGDES="Python bindings for c-ares"
+
+ABTYPE=pep517
```

Comment:
For PEP517 packages, `NOPYTHON2="1"`. cf. https://github.com/AOSC-Dev/autobuild4/blob/ee04025b19be0250947d8605f9e82a964ea0beb8/templates/15-pep517.sh#L17

Reply:
Will handle on Autobuild's end, the warning is pointless since we do not support building Python 2 modules from PEP-517 anyway.